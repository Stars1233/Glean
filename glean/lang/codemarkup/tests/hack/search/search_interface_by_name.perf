{
  "@generated": null,
  "facts_searched": { "hack.SearchInterfaceByName.6": 1 },
  "full_scans": [ ]
}