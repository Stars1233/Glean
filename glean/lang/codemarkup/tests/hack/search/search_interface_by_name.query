query: hack.SearchInterfaceByName { name = "SourceInterface" }
transform: [normord, []]
perf: true
