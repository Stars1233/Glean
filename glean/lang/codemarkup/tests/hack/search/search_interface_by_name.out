[
  "@generated",
  {
    "key": {
      "decl": {
        "container": {
          "interface_": {
            "key": {
              "name": { "key": { "name": { "key": "SourceInterface" } } }
            }
          }
        }
      },
      "name": "SourceInterface"
    }
  }
]